src/components/bcourses/roster/RosterPhotos.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <ul
     class="text-center"
-    style="display: block; overflow: hidden; width: 100%; padding-top: 1rem; margin-left: 1rem"
+    style="display: flex; flex-wrap: wrap; overflow: hidden; width: 100%; padding-top: 1rem; margin-left: 1rem"
```

Comment:
@lyttam - This fix makes sense, however the use of Flexbox might bring back the bad-page-breaks-in-print-mode bug. I believe the @media.page-break CSS is disrupted by flex styles. cc: @mlincoln

Reply:
@lyttam I verified the flex styles didn't break the page-break. Also tweaked the margins so the photos are centered on each page.